interface/src/app/signature/resources.js

```diff
@@ -13,6 +13,19 @@ angular.module('adage.signature.resources', [
 
 .factory('Participation', ['$resource', 'ApiBasePath',
   function($resource, ApiBasePath) {
-    return $resource(ApiBasePath + 'participation/');
+    return $resource(ApiBasePath + 'participation');
```

Comment:
By default the trailing slashes will be removed. So I removed it in the test cases to make it consistent.